wsd/COOLWSD.cpp

```diff
@@ -1182,6 +1182,9 @@ class RemoteConfigPoll : public RemoteJSONPoll
 
     virtual ~RemoteConfigPoll() { }
 
+    // keeps track of remote config layer
+    AutoPtr<AppConfigMap> persistConfig = nullptr;
```

Comment:
Is there a particular reason to have this as a public member *and* not to use the naming convention for member variables?